include/openenclave/attestation/plugin.h

```diff
@@ -320,9 +323,8 @@ oe_result_t oe_unregister_verifier(oe_verifier_t* plugin);
  * @retval OE_NOT_FOUND The plugin does not exist or has not been registered.
  * @retval Otherwise, returns the error code the plugin's function.
  */
-oe_result_t oe_get_evidence(
-    const oe_uuid_t* evidence_format_uuid,
-    uint32_t flags,
+oe_result_t oe_get_evidence_v3(
```

Comment:
Why is this API *not* marked @experimental?   I think it (and its predecessors) should be.